api/app/models/project.rb

```diff
@@ -115,6 +115,14 @@ def to_s
     title
   end
 
+  def updated?
+    updated_at.strftime("%F") != created_at.strftime("%F")
+  end
+
+  def recently_updated?
+    updated? && updated_at >= Time.current - 1.week
```

Comment:
I chose a week, but let me know if there's a different window we want to use for recent.